manim/utils/images.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 import os
 
+from ..config import *
```

Comment:
I would suggest not using `*` here.

Reply:
I"ll change that to import the config dict directly and update many of the docstrings I put on the second line as well any Paramters/Attr mishaps

Reply:
Agree we should not use `*`.

Additionally, please do not import from the config subpackage directly. Instead, import from the top level library: `from manim import config, logger, console` for example.